src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  const visiblePeople: Person[] = people.map(person => {
```

Comment:
Searching for the neede people can be carried out as a separate helper outside the component, this will increase the productivity and readability of the code.